<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Laboratory</class>
 <widget class="QMainWindow" name="Laboratory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laboratory</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>General</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonReset">
         <property name="text">
          <string>Reset Quest</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxControl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Control</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonTuneMachine">
         <property name="text">
          <string>Tune Time Machine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonChargeBattery">
         <property name="text">
          <string>Force Batary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDischarge">
         <property name="text">
          <string>Discharge Battery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRedButton">
         <property name="text">
          <string>Press RED button</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOpenSmallDoor">
         <property name="text">
          <string>Unlock cage door</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCloseSmallDoor">
         <property name="text">
          <string>Lock cage door</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOpenDoor">
         <property name="text">
          <string>Open main door</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTM">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Time machine</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineYear"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Month</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineMonth"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Day</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineDay"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineTime"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="buttonSetTM">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Figures</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="checkBoxT">
            <property name="text">
             <string>Triangle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxS">
            <property name="text">
             <string>Squere</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBosC">
            <property name="text">
             <string>Circle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSetFigures">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Main light</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="buttonLightOn">
            <property name="text">
             <string>On</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonLightOff">
            <property name="text">
             <string>Off</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Figure box</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonBoxOn">
            <property name="text">
             <string>On</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBoxOff">
            <property name="text">
             <string>Off</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBoxOpen">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBoxClose">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
